code/style.css

```diff
@@ -1,4 +1,160 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: sans-serif;
+}
+
+.body-container {
+width: 1400px;
+height: 100%;
+}
+
+.header-container {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.right-header {
+  display: none;
+}
+
+.main-news {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: 1em 0 1em 0;
+}
+
+.left-main img {
+  width: 100%;
+  height: 100%;
+}
+
+.left-header {
+  display: flex;
+  justify-content: center;
+}
+
+.right-main p{
+  font-size: 12px;
```

Comment:
Have a look at the text for the mobile version. It is cut off at the edges.One solution can be to add: 
`margin-left: 50px; `
`width: 80%; `
`text-align: justify; `
But there are probably many other better solutions aswell :)

Reply:
thank you for noticing this, i missed it completely! it is now adjusted to display it correctly. The problem was I had a fixed value on parent. I changed margin: 0 1em 0 1em; to margin: auto 0 1em; to display it correcly.